src/commands/bot-configs/list.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import chalk from 'chalk'
+import {flags} from '@oclif/command'
+import makeDebug from 'debug'
+
+import * as BotConfigsAPI from '../../mix/api/bot-configs'
+import * as MixFlags from '../../utils/flags'
+import MixCommand from '../../utils/base/mix-command'
+import {BotConfigsListParams, MixClient, MixResponse, MixResult} from '../../mix/types'
+import {DomainOption} from '../../utils/validations'
+
+const debug = makeDebug('mix:commands:bot-configs:list')
+
+export default class BotConfigsList extends MixCommand {
+  static description = `list application configurations for available bot.
```

Comment:
Change first line of description to "list application configurations for a bot" with no period at the end.